tests/pytorch/test_pytorch_autolog.py

```diff
@@ -59,7 +61,10 @@ def test_pytorch_autolog_logs_expected_data(pytorch_model):
 
     # Testing optimizer parameters are logged
     assert "optimizer_name" in data.params
-    assert data.params["optimizer_name"] == "Adam"
+
+    # In pytorch-lightning >= 1.1.0, optimizer names are prefixed with "Lightning".
+    prefix = "Lightning" if LooseVersion(pl.__version__) >= LooseVersion("1.1.0") else ""
+    assert data.params["optimizer_name"] == prefix + "Adam"
```

Comment:
Related PR in the pytorch-lightning repo: https://github.com/PyTorchLightning/pytorch-lightning/pull/4658